code/modules/mapping/space_management/space_reservation.dm

```diff
@@ -52,6 +52,12 @@
 	for(var/turf/reserved_turf as anything in release_turfs)
 		SEND_SIGNAL(reserved_turf, COMSIG_TURF_RESERVATION_RELEASED, src)
 
+		// immediately disconnect from atmos
+		reserved_turf.blocks_air = TRUE
+		for(var/turf/atmos_neighbor as anything in reserved_turf.atmos_adjacent_turfs)
+			atmos_neighbor.atmos_adjacent_turfs -= reserved_turf
+		reserved_turf.atmos_adjacent_turfs?.Cut()
+
```

Comment:
similarly inappropriate, use `CALCULATE_ADJACENT_TURFS()` if we are not already hitting this with .empty()